airflow/opt/providers/etna/etna/etls/metis_linker.py

```diff
@@ -129,12 +132,16 @@ def data_frame_update(self, model_name, script, tail, update, metis, model):
             else:
                 data = data.set_index(id, drop=True)
                 for name, attributes in data.T.to_dict().items():
-                    if not self.get_identifier(model_name, name):
+                    if model_name in self.rules and not self.get_identifier(model_name, str(name)):
```

Comment:
Does not filter rows in an upload if there is no gnomon rule for the model.

Reply:
Perhaps I'm not following fully but I think this should be a `not` + `or`  rather than this `and`?

I will test... but just doing code review at the moment.

Reply:
It is like, "only continue (i.e., skip the row) if there is a rule in gnomon AND the identifier does not match the rule", so if there is no rule in gnomon, the row/identifier is assumed to be ok (i.e. it will be validated by magma). we could probably skip this check entirely actually and just put the whole thing on magma...

Reply:
Hmmm.  I like possibility of removing, but the "server error" message that'd often gets produced for bad updates is frustratingly under-informative, and AFAIK, the entire update set, of remainder of the update set, would fail.  So I like handling this here for now to increase the chance of successful updates!